Tests/UnitTests/FoundationExtensions/LocaleExtensionsTests.swift

```diff
@@ -33,15 +33,40 @@ class LocaleExtensionsTests: TestCase {
     }
 
     func testMissingLanguageCode() {
-        expect(Locale(identifier: "").rc_languageCode).to(beNil())
+        // This difference is with running iOS 16 tests with Xcode 15
+        // and iOS 15 tests with Xcode 14
+        #if swift(>=5.9)
+            if #available(iOS 17.0, macOS 14.0, tvOS 17.0, watchOS 10.0, *) {
+                // Fixed in iOS 16 on Xcode 15
```

Comment:
Is this comment supposed to be iOS 17? (Ditto in `testRemovingRegion()` below.)
```suggestion
                // Fixed in iOS 17 on Xcode 15
```

Reply:
It definitely is 😅